scripts/configurator/parser/cmake_parser.py

```diff
@@ -1,23 +1,30 @@
 from cmake_row import cmake_row, row_type
-from typing import List
+from typing import Deque, List
+from collections import deque
 
 def read_file(filepath):
    with open(filepath, 'r') as f:
       return f.readlines()
 
 def parse_rows(file_lines: List[str]) -> List[cmake_row]:
    cmake_rows: List[cmake_row] =  []
-   comment = ""
+   comment_list: Deque[str] = deque()
    for i, line in enumerate(file_lines):
       current_row = cmake_row(line, i)
       cmake_rows.append(current_row)
       if current_row.row_type == row_type.VARIABLE:
          j = i-1
          while j >= 0 and cmake_rows[j].row_type == row_type.SLASH_COMMENT:
-            comment = cmake_rows[j].get_val() + comment
+            comment_list.appendleft(cmake_rows[j].get_val() + "\n")
```

Comment:
**BUG:** now that we have a list of comment rows, it doesn't make any sense to keep "\n".

Reply:
That's actually true, you're totally right